lib/saml_idp/version.rb

```diff
@@ -1,4 +1,4 @@
 # encoding: utf-8
 module SamlIdp
-  VERSION = '0.8.0-18f'.freeze
+  VERSION = '0.8.1-18f'.freeze
```

Comment:
maybe a nit....but this is more than a patch change right? We're changing the number of required arguments to a constructor, so maybe for semver do we need a bigger bump?

I am also OK with just leaving this as-is because realistically we are the only consumers of this fork

Reply:
yeah, I almost think we need to add a fourth digit to indicate we made an update but leave the first three version digits at whatever they were when we forked?

Reply:
Looking back, it appears we've been bumping the minor in the past, so I just bumped it to 0.9.0-18F